stdlib/winreg.pyi

```diff
@@ -1,7 +1,7 @@
 import sys
 from _typeshed import Self
 from types import TracebackType
-from typing import Any, Type, Union
+from typing import Any, Literal, Type, Union
 from typing_extensions import final
```

Comment:
```suggestion
from typing import Any, Type, Union
from typing_extensions import Literal, final
```

Reply:
Yes, thanks you! Python3.10 / 3.11 makes me way too optimistic 😆